programs/building/scarpet_edit_installer.sc

```diff
@@ -0,0 +1,70 @@
+// Scarpet app to download the Scarpet Edit (https://github.com/Ghoulboy78/Scarpet-edit) and its translations.
+// This app is not intended to be downloaded manually, and will only work if downloaded using the "/script download building/scarpet_edit_installer.sc" command
+// to download it from the app store.
+
+printf(text) -> (
+    print(format(text));
+);
+
+checkText = 'Check the Scarpet Edit license at https://github.com/Ghoulboy78/Scarpet-edit/blob/master/licence.txt';
+global_langs = ['es_la', 'it_it'];
```

Comment:
IMO there should be a better way of accessing the languages which are available, maybe by seeing all the files in the release itself using GithubAPI? Tho that would require adding HTML querying to scarpet API (which isn't entirely out of question tbh...)

Reply:
And to prove the point, I made this: https://github.com/gnembon/fabric-carpet/pull/990

Reply:
Well never mind that...